src/config/redis-info.ts

```diff
@@ -6,7 +6,14 @@ export const getRedisInfo = (connectionName: string): IORedis.RedisOptions => ({
 	port: Number(envVars.REDISX_CACHE_PORT) || 6379,
 	host: envVars.REDISX_CACHE_HOST || "127.0.0.1",
 	db: 0,
+	reconnectOnError(err) {
+		const targetError = "READONLY";
 
+		if (err.message.includes(targetError)) {
+			return 1;
+		}
+		return false;
+	},
 	// TODO find out if we still need these options
```

Comment:
They put a fix in on that github link, but people are still complaining, will investigate further outside this PR